natter-docker/minecraft/cf-srv.py

```diff
@@ -124,13 +124,13 @@ def _update_a_record(self, zone_id, rec_id, name, ipaddr, proxied=False, ttl=120
         )
         return data["result"]["id"]
 
-    def _find_srv_record(self, zone_id, name):
+    def _find_srv_record(self, zone_id, name, service, protocol):
         name = name.lower()
         data = self._url_req(
             f"https://api.cloudflare.com/client/v4/zones/{zone_id}/dns_records"
         )
         for rec_data in data["result"]:
-            if rec_data["type"] == "SRV" and rec_data["data"]["name"] == name:
+            if rec_data["type"] == "SRV" and rec_data["name"] == f"{service}{protocol}.{name}":
```

Comment:
这里应当是 `f"{service}.{protocol}.{name}"`，少了一个点。
```suggestion
            if rec_data["type"] == "SRV" and rec_data["name"] == f"{service}.{protocol}.{name}":
```